src/main/resources/db/migration/V19__oppgave.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE oppgave (
+    id              UUID PRIMARY KEY,
+    behandling_id   UUID REFERENCES behandling (id),
+    gsak_oppgave_id BIGINT       NOT NULL,
+    type            VARCHAR      NOT NULL,
+    er_ferdigstilt  BOOLEAN      NOT NULL,
+    opprettet_av    VARCHAR      NOT NULL DEFAULT 'VL',
+    opprettet_tid   TIMESTAMP(3) NOT NULL DEFAULT LOCALTIMESTAMP,
+    endret_av       VARCHAR      NOT NULL,
+    endret_tid      TIMESTAMP(3) NOT NULL DEFAULT LOCALTIMESTAMP
+);
```

Comment:
Her kan det være fint å ha med en index på behandlingId for raskere oppslag :) 
```suggestion
);

CREATE INDEX ON oppgave(behandling_id);
```

Reply:
🏅